### DISCUSSION TITLE
Mahalanobis implementation?

### DISCUSSION THREAD

**Discussion by apiszcz** [intel/scikit-learn-intelex#485] [Ideas]
Are there any plans for a daal4py Mahalanobis implementation that would operate on numpy arrays?
Thank you.

**Comment by PetrovKP** [comment]
Hi! What is this metric interested for you? Support for algorithms such as KNN? Or just a computation?

**Comment by apiszcz** [comment]
Support for algorithms, this specific calculation is the majority of the time for certain filters. Thank you.

**Comment by PetrovKP** [comment]
Are there examples of algorithms that are interesting? Or some scripts? The main problem is that we are computed distances on the side [oneDAL](https://github.com/oneapi-src/oneDAL). And using numpy just won't work. :) If you share cases where it is needed, it will be easier for us to make a decision on extend a libraries to compute this metric.
